lib/bolt/analytics.rb

```diff
@@ -72,7 +72,9 @@ def self.config_path(logger)
 
     def self.load_config(filename, logger)
       if File.exist?(filename)
-        YAML.load_file(filename)
+        data = YAML.load_file(filename)
```

Comment:
Can we use `Bolt::Util.read_optional_yaml_hash`?

Reply:
... ya. Updated. :D

Reply:
That will still raise an error if the file has _invalid_ content, but that seems okay to me.